helm/default-apps-aws/values.yaml

```diff
@@ -163,7 +163,7 @@ apps:
     namespace: kube-system
     # used by renovate
     # repo: giantswarm/cluster-autoscaler-app
-    version: 1.27.3-gs3
+    version: 1.27.3-gs7
```

Comment:
From looking into the changelog, isn't the latest version `...-gs8` (incl. proxy settings)?

Reply:
that was released afterwards, but I bumped it anyway